### PR TITLE
Feat: create a dashboard banner to enable user accounts with SIWE

### PR DESCRIPTION
**Pull request by jmealy**
## What it solves
Adds a banner which for now shows up whenever you have connected your wallet. Later it will only be shown to people who have chosen not to create an account

## Screenshots
image

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/services/siwe/index.ts [hunk 1]
```diff
@@ -38,4 +40,28 @@ Issued At: ${message.issuedAt.toISOString()}`
...
+  return fetch(`${GATEWAY_URL_STAGING}/v1/accounts/${address}`, { credentials: 'include' }).then((res) => {
+    if (res.ok && res.status === 200) {
+      return res.json()
+    } else if (res.status === 404) {
+      return createUserAccount(address)
+    } else {
+      throw new Error('Error fetching User Account')
+    }
+  })
+}
```
Comment (usame-algan) [inline src/services/siwe/index.ts:67]: We can use the latest version 3.22.1 of the typescript-gateway-sdk which contains these functions.

Comment (jmealy) [reply src/services/siwe/index.ts:67]: oh nice, thanks 👍

**Review by usame-algan — APPROVED** [review]

**Comment by katspaugh** [comment]
@jmealy can you move this banner from the actual dashboard to a Storybook story?

File: src/features/siweAccounts/components/EnableAccountBanner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      await signInWithEthereum(provider)
+      account = await getAccount(address)
+    } catch (error) {
+      console.log(error)
+    }
+    if (!account) {
+      try {
+        account = await createAccount({ address: address as `0x${string}` })
+      } catch (error) {
+        console.log(error)
```
Comment (katspaugh) [inline src/features/siweAccounts/components/EnableAccountBanner/index.tsx:31]: We don't console.log errors directly, there's a logError util.

File: src/services/exceptions/ErrorCodes.ts [hunk 3]
```diff
@@ -22,6 +22,7 @@ enum ErrorCodes {
 
   _400 = '400: Error requesting browser notification permissions',
   _401 = '401: Error tracking push notifications',
+  _404 = '401: Error tracking push notifications',
```
Comment (katspaugh) [inline src/services/exceptions/ErrorCodes.ts:25]:
```suggestion
```

Comment (jmealy) [reply src/services/exceptions/ErrorCodes.ts:25]: Oops, thanks!

**Review by katspaugh — APPROVED** [review]
LGTM! Please feel free to merge directly as it doesn't affect any production components.

**Pull request merged**